vendor/github.com/xenserver/go-xenserver-client/vm.go

```diff
@@ -521,6 +521,21 @@ func (self *VM) SetIsATemplate(is_a_template bool) (err error) {
 	return
 }
 
+func (self *VM) GetOtherConfig() (other_config map[string]string, err error) {
```

Comment:
I send a new pull request for go-xenserver-client. but travis ci failed. Could you check about that?